src/lib/scss/private/components/_buttons.scss

```diff
@@ -59,7 +59,8 @@
 
     &:focus {
       outline: none;
-      box-shadow: 0 0 0 2px rgba($color: $ds-primary-500, $alpha: 0.5);
+      box-shadow: 0 0 0 1px white,
```

Comment:
I couldn't make the outline offset color to inherit the background color. I also checked and this is the default for the tailwindcss classes that do this. If anyone knows another way, let me know

Reply:
Transparent doesn't work, because it shows the color of the outline. Basically what happens here is we have one outline on top of the other to mimic an offset